code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -259,38 +259,40 @@
 	item_state = "medpacket"
 	cigarette_type = /obj/item/clothing/mask/cigarette/medical_marijuana
 
-
 /obj/item/storage/fancy/cigarettes/cigpack_uplift
 	name = "\improper Uplift Smooth packet"
-	desc = "Your favorite brand, now menthol flavored."
+	desc = "Your favorite brand, now menthol flavored. To be honest you barely heard of these ones, the packet smell nice though."
 	icon_state = "upliftpacket"
 	item_state = "upliftpacket"
 	cigarette_type = /obj/item/clothing/mask/cigarette/menthol
 
 /obj/item/storage/fancy/cigarettes/cigpack_robust
-	name = "\improper Robust packet"
-	desc = "Smoked by the robust."
+	name = "\improper Robust Red packet"
+	desc = "Smoked by the robust... and these are the all new Robust Reds! What is so new about them? No idea. The second most common cigarettes on the market. The packet smells like robustness ...maybe it's just tobacco."
 	icon_state = "robustpacket"
 	item_state = "robustpacket"
+	cigarette_type = /obj/item/clothing/mask/cigarette/robust
 
 /obj/item/storage/fancy/cigarettes/cigpack_robustgold
 	name = "\improper Robust Gold packet"
-	desc = "Smoked by the truly robust."
+	desc = "Smoked by the truly robust... and snobs who need their cigarettes to be made from actual gold."
 	icon_state = "robustgpacket"
 	item_state = "robustgpacket"
 	cigarette_type = /obj/item/clothing/mask/cigarette/robustgold
 
 /obj/item/storage/fancy/cigarettes/cigpack_carp
 	name = "\improper Carp Classic packet"
-	desc = "Since 2313."
+	desc = "Since 2313. A label on the packaging reads, \"Does not contain Carpotoxin\"... that makes you wonder if they contain carpotoxin. "
```

Comment:
```suggestion
	desc = "Since 2313. A label on the packaging reads, \"Does not contain Carpotoxin\"... that makes you wonder if they contain carpotoxin."
```